justfile

```diff
@@ -0,0 +1,40 @@
+prepare:
+    cp -R docs/pytket-docs-theming/_static docs
+    cp docs/pytket-docs-theming/conf.py docs
+    cp docs/pytket-docs-theming/pyproject.toml docs
+    cp docs/pytket-docs-theming/uv.lock docs
+
+PROJECT_NAME := `(basename $(pwd))`
+
+install: prepare
+    cd docs && uv sync && uv pip install ../.
+
+build *SPHINX_ARGS: install
+    cd docs && uv run sphinx-build {{SPHINX_ARGS}} -b html . build -D html_title={{PROJECT_NAME}}
+
+linkcheck: install
+    cd docs && uv run sphinx-build -b linkcheck . build 
+
+coverage: install
+    cd docs && uv run sphinx-build -v -b coverage . build/coverage 
+
+build-strict: install
```

Comment:
This is the strict recipe that will run in C.I. It will fail on warnings, broken links and bad coverage.